src/libs/actions/Task.js

```diff
@@ -568,20 +568,21 @@ function clearOutTaskInfoAndNavigate(reportID) {
 /**
  * Get the assignee data
  *
- * @param {Object} details
+ * @param {string} assigneeAccountID
```

Comment:
It's Number I guess, can you confirm thanks!

Reply:
It's number, please address this otherwise we are good to go here.
<img width="257" alt="Screenshot 2023-08-01 at 1 55 39 AM" src="https://github.com/Expensify/App/assets/85645967/91aba007-5aad-4e32-a1ba-8036baa2ba00">